spec/integration/api/post_interactions_controller_spec.rb

```diff
@@ -407,7 +407,7 @@
       post(
         api_v1_post_vote_path("999_999_999"),
         params: {
-          poll_answer_id: @poll_answer.id,
+          poll_answer: @poll_answer.id,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.